BraveRewards/Ledger/BATBraveLedger.h

```diff
@@ -88,17 +88,19 @@ NS_SWIFT_NAME(BraveLedger)
 - (void)activityInfoWithFilter:(nullable BATActivityInfoFilter *)filter
                     completion:(void (^)(BATPublisherInfo * _Nullable info))completion;
 
-// Unavailable until we figure out what it actually does and how to use it
 - (void)publisherActivityFromURL:(NSURL *)URL
                       faviconURL:(NSURL *)faviconURL
                         windowID:(uint64_t)windowID
```

Comment:
We should just remove this argument from the public API, since I thought we were always going to send in 1 to the actual ledger API?